### ISSUE TITLE
PointSize is not handled in 3D

### ISSUE THREAD

**Issue by rocky** [Mathics3/mathics-django#62]
Try 
```
Table[Graphics3D[{PointSize[r], Point[{0, 0, 0}]}], {r, {0.02, 0.05, 
   0.1, 0.3}}]
```

Recently in https://github.com/mathics/Mathics/pull/1411 field member `pointSize` was added to the [data that is passed in json](https://github.com/mathics/Mathics/blob/b8968a0c3a6ffb1f34484d4c31d6b0068888a450/mathics/formatter/json.py#L73). 

@TiagoCavalcanteTrindade is this somethone you'd be interested in? 

(There are also problems in aymptote for 2D and PlotStyle, but those will be handled separately in the Mathics core project.

**Comment by TiagoCavalcanteTrindade** [comment]
@rocky I think it'll be easy to do.

**Comment by rocky** [comment]
@TiagoCavalcanteTrindade  Wow - that was fast. However the pionts look like black squares (not boxes because they don't rotate) rather than round points. 

Could that be addressed? 

![pointsize](https://user-images.githubusercontent.com/8851/121052364-4e6a4880-c788-11eb-82f9-4cbe367f45f3.png)

**Comment by TiagoCavalcanteTrindade** [comment]
@rocky I will see if I can fix this.

**Comment by rocky** [comment]
Fixed by #63
